<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnGetFile">
          <property name="text">
           <string>录音输出文件:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOutFile"/>
        </item>
        <item>
         <widget class="QLabel" name="labelRecordDuration">
          <property name="text">
           <string>已录制18秒</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>录音设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>输入设备</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="comboBoxInputDev">
                 <property name="minimumSize">
                  <size>
                   <width>250</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>音频编码</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="comboBoxCode"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>采样率</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="comboBoxSamplingRate"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>通道数</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="comboBoxChannalCount"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>编码模式</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButtonFixedQuality">
                    <property name="text">
                     <string>固定品质</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSlider" name="horizontalSliderFixedQuality">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QRadioButton" name="radioButtonFixedBitRate">
                    <property name="text">
                     <string>固定比特率</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="comboBoxFixedBitRate"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>探测QAudioBuffer信息</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>缓冲区字节数</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="spinBoxBufByteCount">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>150</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>缓冲区时长</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QSpinBox" name="spinBoxBufDuration"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>缓冲区帧数</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="spinBoxFrameNumber"/>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>采样率</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QSpinBox" name="spinBoxSamplingRate"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>QAudioFormat信息</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>channolCount</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBoxChannolCount"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>byteOrder</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEditByteOrder"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>code</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEditCode"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>sampleType</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEditSampleType"/>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>sampleSize</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QSpinBox" name="spinBoxSampleSize"/>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>sampleRate</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1">
                   <widget class="QSpinBox" name="spinBoxSampleRate"/>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>byterPerFrame</string>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="1">
                   <widget class="QSpinBox" name="spinBoxBytePerFrame"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecord"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecord">
   <property name="text">
    <string>录音</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
